doc/nrf/protocols.rst

```diff
@@ -4,7 +4,7 @@ Protocols
 #########
 
 The following user guides describe the supported protocols.
-They introduce you to concepts that are important to work with the protocol and guide you through developing your application.
+They introduce you to concepts that are important to work with the protocol and guide you through developing your application. For Wi-Fi coexistence, see the :ref:`Adding Wi-Fi Coexistence support to short-range radio applications <ug_radio_coex>` page.
```

Comment:
Wi-Fi coexistence is an imprecise term. I think it should be "Wi-Fi coexistence with short range protocols (Bluetooth LE, Thread etc.)".

Also I think this sentence is a bit misplaced and hard to see. I think it would be best to put it in the below tree next to Wi-Fi if only this is possible.

Reply:
I don't think it's possible to add rst subpages with custom names/sentences in their names. However, I think maybe if we swap the order of the last few items (make it Thread, Zigbee, then Wi-Fi as the last item) this sentence could be put right below the tree, making it directly follow the Wi-Fi subpage.